### PR TITLE
Test 1: Send node_id to ci and verify we got it and it's not null

### PR DESCRIPTION
**Pull request by mosheka**
Adding (or actually in this phase replacing test 0 with) test 1
We will be able to combine test 0, when we implement either: down stream topic, or when adding environment per test

### CODE REVIEW

File: tests/base_test_agent.py [hunk 1]
```diff
@@ -12,6 +14,19 @@ def send(self, data):
...
+                break;
+        return node_id
+
+
 if __name__ == '__main__':
     t = BaseDevnetAgent()
-    t.send('UP')
\ No newline at end of file
+#    t.send('UP')
+    t.send(t.get_node_id())
```
Comment (gavra14) [inline tests/base_test_agent.py]: Is it a valid behaviour to send NULL?

File: tests/test_1.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import config
+import os
+import time
+from google.cloud import pubsub_v1
+import unittest
+
+class Test1(BaseTest):
+    def test_sendId(self):
+        self.subscriber.subscribe(self.subscription_path, callback=self.callback)
+        for i in range(0, self.testLen):
```
Comment (gavra14) [inline tests/test_1.py]: As we discussed, the "logic" of the verification should be decoupled from the test itself (maybe through another object) so you won't duplicate the same logic.

File: Makefile [hunk 3]
```diff
@@ -23,12 +23,17 @@ install:
 	@make build
 
 run:
-	@rm -rf $(shell pwd)/test.log
+	@mkdir $(shell pwd)/logs || echo ''
```
[the remark this answers is not in the document]
Comment (mosheka) [reply Makefile]: Done

File: Makefile [hunk 4]
```diff
@@ -23,12 +23,17 @@ install:
...
+	@rm -rf $(shell pwd)/logs/*.log
 	@if [ ! "$(sudo docker network ls | grep devnet)" ]; then sudo docker network create devnet || true; fi
 	@(docker stop agent && docker rm agent) || echo "" &&\
-	docker run --network=devnet --name agent -p 8080:8080 -e PUBSUB_VERIFICATION_TOKEN='1234' -e PUBSUB_TOPIC='topic' -e GOOGLE_CLOUD_PROJECT='spacemesh-198810' -v $(shell pwd)/tests:/opt/devnet spacemesh/devnet_agent:latest python3 /opt/devnet/base_test_agent.py  >> $(shell pwd)/test.log 2>&1 &
+	docker run --network=devnet --name agent -p 8080:8080 -e PUBSUB_VERIFICATION_TOKEN='1234' -e PUBSUB_TOPIC='topic' -e GOOGLE_CLOUD_PROJECT='spacemesh-198810' -v $(shell pwd)/tests:/opt/devnet -v $(shell pwd)/logs:/opt/logs spacemesh/devnet_agent:latest python3 /opt/devnet/base_test_agent.py >> $(shell pwd)/logs/test.log 2>&1 &
```
[the remark this answers is not in the document]
Comment (mosheka) [reply Makefile]: Seems like $(CURDIR) is not standard

File: Makefile [hunk 5]
```diff
@@ -23,12 +23,17 @@ install:
...
+	@(docker stop node && docker rm node) || echo "" &&\
+	docker run --network=devnet --name node -p 7513:7513 spacemesh/node:latest /go/src/github.com/spacemeshos/go-spacemesh/go-spacemesh >> $(shell pwd)/logs/node.log 2>&1 &
 	@(docker stop test_server && docker rm test_server) || echo "" &&\
```
[the remark this answers is not in the document]
Comment (mosheka) [reply Makefile]: done

File: tests/base_test_agent.py (hunk 1 above)
Comment (mosheka) [reply tests/base_test_agent.py]: NULL is just a string. 
I believe it's not a valid ID of something like that
You cannot send empty strings in PubSub

And last, we'll probably enlarge the message, and create somekind of protocol here :-)

File: tests/test_0.py [hunk 6]
```diff
@@ -10,9 +10,10 @@ def test_verifyUp(self):
         self.subscriber.subscribe(self.subscription_path, callback=self.callback)
         for i in range(0, self.testLen):
             if self.endFlag:
-                self.assertEqual(b'UP', self.message)
                 return
```
[the remark this answers is not in the document]
Comment (mosheka) [reply tests/test_0.py]: done

**Pull request merged**